examples/models/llama/export_llama_lib.py

```diff
@@ -519,7 +523,53 @@ def canonical_path(path: Union[str, Path], *, dir: bool = False) -> str:
         return return_val
 
 
+def download_and_convert_hf_checkpoint(modelname: str) -> str:
```

Comment:
Put this in a separate util file? I think it'll be useful elsewhere too potentially and also this file is getting quite bloated too i think.